homeassistant/components/peco/sensor.py

```diff
@@ -57,17 +64,24 @@ async def async_update_data():
 
     await coordinator.async_config_entry_first_refresh()
 
-    _LOGGER.info("Setting up sensor platform")
     county = conf["county"]
-    _LOGGER.info("County: %s", county)
+
+    county_name = county.lower()[0].upper() + county.lower()[1:]
+
+    sensors = []
+    for sensor in SENSOR_LIST:
+        sensor_name = getattr(sensor, "name", "This should never happen").format(
+            county_name
+        )
```

Comment:
just define the Entity.name attribute in `__init__()` with an f-string instead.
`self._attr_name = f"{country} {description.name}"`